src/api/helpers/get-expiring-applications/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Application, Context } from '.keystone/types'; // eslint-disable-line
+import { APPLICATION } from '../../constants';
+import { dateInTheFutureByDays } from '../date';
+
+const { IN_PROGRESS } = APPLICATION.STATUS;
+
+const { REMINDER_DAYS, EARLIER_LIMIT_TIME_SET, LATER_LIMIT_HOURS, LATER_LIMIT_TIME_SET } = APPLICATION.SUBMISSION_DEADLINE_EMAIL;
+
+/**
+ * getExpiringApplications
+ * gets expiring applications - submissionDeadline in 2 days time
+ * returns array of applications
+ * @param {Context} context
+ * @returns {Promise<Application[]>} Array of applications
+ */
+const getExpiringApplications = async (context: Context): Promise<Application[]> => {
+  try {
+    console.info('Getting expiring applications - getExpiringApplications helper');
+
+    const today = new Date();
+
+    const twoDaysTime = dateInTheFutureByDays(today, REMINDER_DAYS);
+    // midnight 2 days in the future - earliest time of the day
+    const earlierLimit = new Date(twoDaysTime.setHours(EARLIER_LIMIT_TIME_SET, EARLIER_LIMIT_TIME_SET, EARLIER_LIMIT_TIME_SET, EARLIER_LIMIT_TIME_SET));
+    // latest time in the day
+    const laterLimit = new Date(twoDaysTime.setHours(LATER_LIMIT_HOURS, LATER_LIMIT_TIME_SET, LATER_LIMIT_TIME_SET, LATER_LIMIT_TIME_SET));
```

Comment:
Finding this hard to read, can we add a space after each comment

Not sure how this could be improved to be easier to read or less DRY e.g one call to `new Date` has the same `EARLIER_LIMIT_TIME_SET` param being passed 4 times

Maybe new helper functions?

Reply:
changed